packages/docs/src/routes/docs/(qwik)/components/state/index.mdx

```diff
@@ -428,10 +428,10 @@ export default component$(() => {
     count: number;
   }>(async ({ track, cleanup }) => {
     track(() => name.value);
-    const abortController = new AbortController();
-    cleanup(() => abortController.abort('cleanup'));
+    const controller = new AbortController();
+    cleanup(() => controller.abort());
```

Comment:
I don't understand what the problem is - is it because of the 'cleanup' string?

Reply:
I don't think these changes actually fix the issue as they are not related to the demo that fails.

Have a look at this PR instead: https://github.com/QwikDev/qwik/pull/6893.

Reply:
> I don't understand what the problem is - is it because of the 'cleanup' string?

Yes because of the ‘cleanup’ string. It's weird but I'm still digging into it

Reply:
> I don't think these changes actually fix the issue as they are not related to the demo that fails.
> 
> 
> 
> Have a look at this PR instead: https://github.com/QwikDev/qwik/pull/6893.
> 
> 
> 
> 

You can try it by my code if you don't believe that can fix it.
And I have already shown my picture that everything is ok.

Reply:
This issue doesn't occur locally, that's why the screenshots show that everything is ok.

Everything works locally even with the `clean up` string passed to the abort controller.

Reply:
you are right, my bad. 
I will close this PR.